pkg/driver/controller.go

```diff
@@ -550,10 +573,6 @@ func (d *Driver) ControllerExpandVolume(ctx context.Context, req *csi.Controller
 	if capRange == nil {
 		return nil, status.Error(codes.InvalidArgument, "Capacity range not provided")
 	}
-	if capRange.GetLimitBytes() > 0 && capRange.GetRequiredBytes() > capRange.GetLimitBytes() {
```

Comment:
Can you remind me why we are removing this again? I believe it was because Kubernetes doesn't support Limit. Isn't the CSI driver specification independent of Kubernetes although? Maybe we should keep this in unless this check already occurs under the hood.

Reply:
The external provisioner doesn't actually support `LimitBytes`, it only ever supplies the `RequiredBytes`:   
https://github.com/kubernetes-csi/external-provisioner/blob/7eed97a6c45d9542f097adaed0c7c38becd7f0db/pkg/controller/controller.go#L643-L644

Same with the external resizer: https://github.com/kubernetes-csi/external-resizer/blob/990c87e85d9dc7bf4b554cb54b58455628c17cd5/pkg/csi/client.go#L143. As such, there's no point in having this check right now.